tests/shared/src/test/scala/org/http4s/websocket/WebSocketFrameAggregatorSuite.scala

```diff
@@ -0,0 +1,160 @@
+/*
+ * Copyright 2013 http4s.org
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.http4s.websocket
+
+import cats.effect.SyncIO
+import fs2.Stream
+import org.http4s.Http4sSuite
+import org.http4s.websocket.WebSocketFrame.Binary
+import org.http4s.websocket.WebSocketFrame.Close
+import org.http4s.websocket.WebSocketFrame.Continuation
+import org.http4s.websocket.WebSocketFrame.Ping
+import org.http4s.websocket.WebSocketFrame.Text
+import scodec.bits.ByteVector
+
+import java.nio.charset.StandardCharsets.UTF_8
+
+class WebSocketFrameAggregatorSuite extends Http4sSuite {
```

Comment:
Sorry if I missed it, can we add some tests to establish the behavior for invalid sequences? e.g. missing continuations, wrong type, etc.